### DISCUSSION TITLE
rbenv install 3.3.5 fails on WSL with windows 11

### DISCUSSION THREAD

**Discussion by Adrenno** [rbenv/ruby-build#2442] [Build failures]
![Screenshot 2024-09-07 120508](https://github.com/user-attachments/assets/7d3187a3-d486-47b9-a9d5-4009ddabac47)
i'm new to this thing and have no idea how to fix this. pls help and ask for anything you need to know

**Comment by mislav** [comment]
Hi, when you install software using a system package manager like apt-get, you typically need admin (also called "root") privileges. An easy way to give a command root privileges is to invoke a command by prefixing it with "sudo".

You only need the "sudo" prefix for apt-get commands, but you do not need to put "sudo" before any "rbenv" commands. Hope this helps!
